### PR TITLE
adding my lab with the horse racing

### PR DESCRIPTION
**Pull request by d3nnyyy**

### CODE REVIEW

File: Race.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include "Race.h"
+#include "RaceHorse.h"
+using namespace std;
```
Comment (Andriy723) [inline Race.cpp:4]: After "RaceHouse.h" there should be an empty line then the namespace std

File: Race.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#ifndef RACE_H
+#define RACE_H
+#include "RaceHorse.h"
```
Comment (Andriy723) [inline Race.h:4]: There must be an empty line before "RaceHouse.h"

File: Race.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#include <list>
```
Comment (Andriy723) [inline Race.h:6]: There must be an empty line after "RaceHouse.h"

File: RaceHorse.h [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#ifndef RACEHORSE_H
+#define RACEHORSE_H
+using namespace std;
```
Comment (Andriy723) [inline RaceHorse.h:4]: After define there should be an empty line

File: RaceHorse.h [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+#include <iostream>
```
Comment (Andriy723) [inline RaceHorse.h:6]: There should be an empty line after namespace std

File: main.cpp [hunk 6]
```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include <list>
+#include <math.h>
```
Comment (Andriy723) [inline main.cpp:3]: There should be an empty line after <math.h>

File: main.cpp [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+        racer.ageDiffer = abs(averageAge - racer.age);
+        cout << racer.name << "'s age difference is  " << racer.ageDiffer << endl;
+    }
+
+    race.sortRacersByAge();
+    cout << "\nAfter sorting:" << endl;
+    for (RaceHorse &racer : race.racerList)
+    {
+
```
Comment (Andriy723) [inline main.cpp]: There should not be an empty line

File: main.cpp [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+        cout << racer.name << "'s age difference is  " << racer.ageDiffer << endl;
+    }
+
+    // sorting them by speed
+
+    cout << "\n\nBefore sorting:" << endl;
+    for (const RaceHorse &racer : race.racerList)
+    {
+
```
Comment (Andriy723) [inline main.cpp]: There should not be an empty line

File: main.cpp [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+        cout << racer.name << "'s speed is  " << racer.speed << endl;
+    }
+    race.sortRacersBySpeed();
+
+    cout << "\nAfter sorting:" << endl;
+    for (const RaceHorse &racer : race.racerList)
+    {
+
```
Comment (Andriy723) [inline main.cpp]: There should not be an empty line

File: main.cpp [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+        cout << racer.name << "'s speed is  " << racer.speed << endl;
+    }
+
+    // determining the winner
+
+    for (const RaceHorse &winner : race.racerList)
+    {
+
```
Comment (Andriy723) [inline main.cpp]: There should not be an empty line

**Review by Andriy723 — CHANGES_REQUESTED** [review]
There are no indents, lowercase methods

**Review by Andriy723 — APPROVED** [review]
All problems that i noticed were resolved. Good job!

File: main.cpp [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+        cout << "\nThe Winner is " << winner.name << endl;
+        break;
+    }
+
+    // assign to the best 3 racers their place in race
+
+    int place = 1;
+    for (RaceHorse &racer : race.racerList)
+    {
+        while (place != 4)
```
Comment (zveres-ss) [inline main.cpp:91]: 4 is magic number, make it as constant with well defined name

**Review by zveres-ss — CHANGES_REQUESTED** [review]

**Pull request merged**